site/content/en/guides/consumer/function/export/tekton/_index.md

```diff
@@ -10,7 +10,9 @@ In this tutorial, you will pull an example blueprint that declares Kubernetes re
 
 ## Before you begin
 
-Before diving into the following tutorial, you may need to create a public repo on GitHub if you don't have one yet, e.g. `function-export-example`.
+*New to Tekton? Here is a [quickstart](https://tekton.dev/docs/getting-started/)*.
```

Comment:
Rest of the kpt docs keep all links at the bottom of the doc which makes it easy to change many links at once. Let's follow this convention.

This applies wherever you've added links.